addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -941,6 +945,19 @@ section, .oe_img_bg, [data-oe-shape-data] {
         -webkit-background-clip: inherit;
         -webkit-text-fill-color: inherit;
     }
+
+    .btn, *[style*=color], *[style*=background-color], .o_record_cover_image {
+        @extend .text-solid;
+    }
+    .fa, .badge, .dropdown-menu {
+        @extend .text-solid;
+        background-image: none;
+    }
+}
+
+.text-solid {
+    -webkit-background-clip: border-box;
+    -webkit-text-fill-color: initial;
```

Comment:
Yes I think this definitely convinces me to not add the gradient on text by default feature if this is actually needed. Won't you agree? :grimacing: